cli/userinput/userinput.go

```diff
@@ -0,0 +1,61 @@
+package userinput
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/open-horizon/anax/cli/cliconfig"
+	"github.com/open-horizon/anax/cli/cliutils"
+	"github.com/open-horizon/anax/policy"
+	"net/http"
+)
+
+//Display a list of the current userInputs of the node
+func List() {
+	var inputs policy.UserInput
```

Comment:
Please change policy.UserInput to []policy.UserInput because of the changes in the recent committed PR.